complaintdatabase/views.py

```diff
@@ -30,10 +28,7 @@ class LandingView(TemplateView):
 
     @property
     def template_name(self):
-        if flag_enabled(self.request, 'MOSAIC_CCDB'):
-            return "landing-page.html"
-        else:
-            return "landing-page-original.html"
+        return "landing-page.html"
```

Comment:
Minor suggestion: this doesn't need to be a property anymore and can just be defined on the class [as documented here](https://docs.djangoproject.com/en/1.8/ref/class-based-views/base/#django.views.generic.base.TemplateView):

```py
class LandingView(TemplateView):
    template_name = 'landing-page.html'

    ...
```